test/utils/e2e-framework-mock-manager.js

```diff
@@ -37,6 +37,9 @@ const setupStripe = () => {
         if (key === 'stripe_connect_account_name') {
             return 'Test Account';
         }
+        if (key === 'theme_session_secret') {
```

Comment:
This should probably be in like `setupMembers` function, but I think this is showing a clear pattern that we need to be able to stub settings before getting an agent/booting Ghost